# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/humantime/bulk.yaml
# Commit: a651d4323066
# Full Hash: a651d43230664837c308663382ccc3bcd7298dc2
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r be008ddb2418 -r a651d4323066 third_party/rust/humantime/bulk.yaml
--- a/third_party/rust/humantime/bulk.yaml	Fri Jan 27 21:25:50 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,8 +0,0 @@
-minimum-bulk: v0.4.5
-
-versions:
-
-- file: Cargo.toml
-  block-start: ^\[package\]
-  block-end: ^\[.*\]
-  regex: ^version\s*=\s*"(\S+)"